libs/utils/android/workloads/gmaps.py

```diff
@@ -90,20 +90,38 @@ def run(self, out_dir, location_search, swipe_count=5, collect=''):
         # Allow the activity to start
         sleep(1)
 
-        self.tracingStart()
-        self._log.info('Opening GMaps to [%s]', loc_url)
-        # Let GMaps zoom in on the location
-        sleep(2)
+        # Hack: screen resolution is memoized, so call it here
```

Comment:
Not too happy with that one here... But I felt the need for a dry-run mode in devlib, as modifying every method in System to add a dry-run seemed absolutely barbaric.